R/gh_whoami.R

```diff
@@ -0,0 +1,74 @@
+#' Info on current GitHub user and token
+#'
+#' Reports wallet name, GitHub login, and GitHub URL for the current
+#' authenticated user, the first few and last characters of the token, and the
+#' associated scopes.
```

Comment:
I don't know, TBH. Showing 8 characters makes the token 2^(8*4) times less secure. We would show 32 bits of the 160.